dor/providers/filesets.py

```diff
@@ -68,18 +74,40 @@ def now():
     return datetime.now().isoformat()
 
 
+PROFILE_COMMANDS: dict[str, list[Command]] = {
+    "compress-source": [Command(operation=CompressSourceImage, kwargs={})],
+    "extract-text": [
+        Command(operation=ExtractImageTextCoordinates, kwargs={}),
+        Command(operation=ExtractImageText, kwargs={}),
+        Command(operation=CreateTextAnnotationData, kwargs={})
+    ],
+    "extract-text-plain": [
+        Command(operation=ExtractImageText, kwargs={}),
+    ],
+    "label-service": [
```

Comment:
If you can imagine a way to make the target based on the file type this operation is attached to, that'd be amazing. But definitely not required for the short term.